website/docs/docs/dbt-cloud/on-premises/mult_tenancy_deployment.md

```diff
@@ -0,0 +1,125 @@
+
+### Create a dbt Cloud Account
+To create a new account, navigate to the dbt Cloud backend at `https://<hostname>/backend`. You can login as the `superuser` to perform the steps below.
+
+After logging in, you can create a new account and invite members of your team, by doing the following:
+
+1. Under `Database` > `Accounts`, click `+ Add`. 
+2. Enter a name for the account
+3. Click `Show` next to `Resources`. Enter how many run slots (concurrent jobs) to allow for this account. 
+    **If you are unsure, pick 1 for now.**
+    
+![image](https://user-images.githubusercontent.com/46451573/81334360-f2f4f500-9073-11ea-9412-e1b5428fff88.png)
+
+4. Setting up SSO. Click `Show` next to `Enterprise options`.
+Fill out this section of the enterprise options with your `unique identifier` and 
+`login slug` for the specific SSO application created above. Once you click `Save`
+at the bottom, the Enterprise login url will be updated.
+
+![image](https://user-images.githubusercontent.com/46451573/81210559-0d5d9e80-8fa0-11ea-9422-eebd834f9e96.png)
+
+5. Add users to the account (this is important -- if you don't add users, nobody will be able to access the account!)
+6. Save the new account.
+
+
+#### Helpful Hints
+
+ - For Azure AD and GSuite, the unique identifier is the deployment id provided when creating the app.
+ - For Okta, the unique identifier is the "Identity Provider Issuer"
+ - The `login slug` is added to the Enterprise Login URL to create a unique login link for users of this account.
+ - **If you aren't sure which values to use, contact your Account Manager.**
+ - It is highly recommended that the first users to associate to the account would be the account administrators. If those users are not already setup in the backend, once you provide the SSO login URL, the user will be created. Alternatively a new user can be emailed an invite via the frontend after configuration is completed. 
+
+
+### Onboarding users
+Here are the steps to onboard users to the new account. 
+
+
+
+##### 1. Create the Projects
+
+The dbt cloud administrator will need to create the projects either themselves or invite an AccountAdmin level user to the account to establish the projects necessary for the required environments. 
+    
+- Remember that it is one data warehouse connection and one git repository per dbt cloud project.
+- This needs to be created on the frontend of dbt Cloud.
+        
+##### 2. Establish the connections to the data warehouse and git repository.
+ This needs to be created on the frontend of dbt Cloud.
+ 
+##### 3. Create the Groups
+
+ - If you want to utilize [Enterprise Permissions](dbt-cloud/dbt-cloud-enterprise/enterprise-permissions.md), create the groups with the appropriate SSO mapping groups so that new users will be associated to the appropriate groups once logged in. If you do 
+ not have SSO mapping groups from your IdP, users can be manually added to groups if not SSO mapping groups are established
```

Comment:
it looks like this sentence got screwed up -- maybe:

> If you have not configured SSO mapping groups from your IdP, you can still manually add users to groups from the dbt Cloud backend.

Reply:
Hmm I debated about this. We want and expect the majority of our enterprise clients to manage groups via their IdP but what if they don't have groups that they want to map to dbt cloud groups? Manually adding users to groups isn't ideal at all imo.